docs/howto/openstack/nova/new-server.md

```diff
@@ -23,90 +23,114 @@ to work with the OpenStack CLI, please do not forget to [source the RC
-    at the top. Something else that affects the cost is the size of the root
-    device. Take a look at the _Size_ parameter below, and notice the
-    default (in
+    Below the region selection is a drop-down menu where you must select a
+    _server profile_. Currently, four profiles are available: _Generic_,
+    _High CPU_, _Low Latency Disk_, and _GPU_. Selecting each of those gives
+    you access to a subset of all available
+    [_flavors_](../../../reference/flavors/index.md). Now, as the profile
+    names suggest, each profile but the _Generic_ points to a set of flavors
+    suitable for servers intended to run specific types of applications.
+    More specifically, the _Generic_ profile is for general-purpose cloud
+    servers, the _CPU_ profile is for servers designed to host CPU-intensive
+    applications, the _Low Latency Disk_ is for servers that boot off of a
+    local low-latency disk, whereas the _GPU_ profile is for servers that
+    boot off of a local SSD disk and, most importantly, have direct access
+    to an NVIDIA vGPU. We should point out that _not_ all profiles may be
+    available in any region. Let us choose the _Generic_ profile for now,
+    which, by the way, is available in any of the regions.
+
+    > Please keep in mind that profile selection is not available via the
+    > OpenStack CLI. When creating servers using the `openstack` client,
+    > though, you can still select flavors suitable for specific types of
+    > workloads.
+
+    ![Create new server](assets/new-server/shot-03.png)
+
+    In the _Boot source_ section below, click the dropdown menu on the left
+    and make sure you select _Image_, so you can choose one of the readily
+    available OS images to boot the new server off of. To pick one of the
+    images, click on the dropdown menu on the right. For this how-to guide,
+    we have chosen _ubuntu_ in general and _Ubuntu 22.04 Jammy Jellyfish_ in
+    particular.
+
+    ![Boot source](assets/new-server/shot-04.png)
+
+    Next up, notice that the _Boot Target_ is preselected for you; depending
+    on the server profile you have already selected, the boot target will be
+    either _Volume_ (_Generic_, _High CPU_, _GPU_ profiles) or _Ephemeral_
+    (_Low Latency Disk_ profile). Regarding the server's CPU core count and
+    amount of memory, set the
+    [_Flavor_](../../../reference/flavors/index.md) accordingly. We suggest
+    selecting a flavor that specifies a server with 1 CPU core and 1GiB of
+    RAM. Please note that, depending on the chosen flavor, the estimated
+    monthly cost of the server changes. (While a server is shut off you are
+    still getting charged for it, but less so.) At any time, this estimated
+    cost is displayed in the green rectangular area at the top. Something
+    else that affects the cost is the size of the root device. Take a look
+    at the _Volume_ parameter below, and notice the default (in
```

Comment:
Here, again, we have a long wall of text that needs breaking up.

Reply:
I'm breaking up (the wall of text, that is).